### PR TITLE
content: Narriative change when first offered the Emerald Sword

### PR DESCRIPTION
**Pull request by dashkal16**
**Content (Artwork / Missions / Jobs)**

## Summary
Regarding the archaeology mission chain to obtain the Emerald Sword.
Specifically the mission `Sheragi Archaeology: The Emerald Sword 2`

This PR contains small change to the conversation on Zug before heading out to retrieve the emerald Sword acknowledging Albert's willingness to simply allow the player to have the ship.

## Problem
This PR is intended as a potential solution to the (in my opinion) slightly odd situation where an archaeologist is happy to simply promise you what feels like a priceless historical find.
The goal is to still give the player the ship, but to make it feel slightly more reasonable in the narrative.

## Save File
I did not create a save file for this specific change.  Instead I wrote a plugin with the mission copied over and adjusted to be offered from New Boston with no requirements (and renamed to avoid a collision).

jobs.txt:
```
mission "Test - Sheragi Archaeology: The Emerald Sword 2"
	landing
	name "Test - The Emerald Sword"
	description "Test - Bring an additional jump drive to Zug."
	source "New Boston"
#	to offer
#		has "Sheragi Archaeology: The Emerald Sword 1: done"
	
	on offer
		...
```

### CODE REVIEW

**Comment by dashkal16** [comment]
This was discussed briefly in the `#content-creating` channel in the discord:
https://discord.com/channels/251118043411775489/266345072554016768/1069410775720005815

One very good question was raised: Should the ship be offered at all here?

**Comment by RisingLeaf** [comment]
Well, hello there, I hope you enjoy your stay at this project.

**Comment by Karirawri** [comment]
You have my blessing. I should have done that myself tbh.

File: data/sheragi/archaeology missions.txt [hunk 1]
```diff
@@ -1280,6 +1280,13 @@ mission "Sheragi Archaeology: The Emerald Sword 2"
 			label return
 			`	"What do I get in return?" you ask.`
 			`	"Fame and fortune, my friend. And if the ship is still there, I think you deserve to keep it," Albert responds. "You've helped me so much, and you went to hell and back to get this black box decoded. Give us some time to assemble a crew to survey the system and help refurbish the ship, and in the meantime, go get us a jump drive. Come back to us once you have it."`
+			choice
+				`   "You're just going to let me have it? Isn't this something of a priceless historical artifact?"`
+					goto askkeepship
+				`   "Thank you."`
+					accept
+			label askkeepship
+			`   Albert responds, "Well, yes, but I'd feel much more comfortable with it in your hands than sitting here on <origin> waiting to be destroyed or stolen in a pirate raid. The millitia works hard, but they can't be everywhere."`
```
Comment (Ferociousfeind) [inline data/sheragi/archaeology missions.txt:1289]: small typo
(am I a reviewer now?)
```suggestion
			`   Albert responds, "Well, yes, but I'd feel much more comfortable with it in your hands than sitting here on <origin> waiting to be destroyed or stolen in a pirate raid. The militia works hard, but they can't be everywhere."`
```

**Review by tibetiroka — APPROVED** [review]
Grammar looks OK.

**Review by roadrunner56 — APPROVED** [review]

**Comment by dashkal16** [comment]
Fixed some lingering spaces that should have been tabs.

File: data/sheragi/archaeology missions.txt [hunk 2]
```diff
@@ -1280,6 +1280,13 @@ mission "Sheragi Archaeology: The Emerald Sword 2"
 			label return
 			`	"What do I get in return?" you ask.`
 			`	"Fame and fortune, my friend. And if the ship is still there, I think you deserve to keep it," Albert responds. "You've helped me so much, and you went to hell and back to get this black box decoded. Give us some time to assemble a crew to survey the system and help refurbish the ship, and in the meantime, go get us a jump drive. Come back to us once you have it."`
+			choice
+				`	"You're just going to let me have it? Isn't this something of a priceless historical artifact?"`
+					goto askkeepship
+				`	"Thank you."`
+					accept
+			label askkeepship
+			`	Albert responds, "Well, yes, but I'd feel much more comfortable with it in your hands than sitting here on <origin> waiting to be destroyed or stolen in a pirate raid. The militia works hard, but they can't be everywhere."`
```
Comment (EjoThims) [inline data/sheragi/archaeology missions.txt:1289]:
```suggestion
			`	Albert responds, "Well, yes, but I'd feel much more comfortable with it in your hands than sitting here on <origin> waiting to be destroyed in loose fire during a pirate raid. Besides that though, while the Free Worlds work hard and have a stronger presence here than the Navy ever did, they can't stop everyone who may have an interest in this ship. At least with you it's a moving target."`
```

Can't you only get this far after completing the main plot?

Even if it is still just the militia though, they don't need to be everywhere to stop this being stolen, just here, so phrasing it that way reads as a cop out to me.

Especially when the real threat to a piece in a guarded museum is most likely not random pirates grabbing whatever they can, but organized attack from specific groups (some of whom also either are or are working with pirates [BBub and Mammon spring readily to mind]) deliberately targeting the ship and/or its tech specifically.

Suggested change also leaves a bit more of an explicit hook that can be capitalized on later if we desire.

**Review by EjoThims — CHANGES_REQUESTED** [review]

File: data/sheragi/archaeology missions.txt (hunk 2 above)
Comment (dashkal16) [reply data/sheragi/archaeology missions.txt:1289]: A previous mission `Sheragi Archaeology: The Box 1` requires `has "main plot completed"`, so I feel safe mentioning the FW militia directly.
Your suggestion is a definitely improvement to my eyes.  Thank you!

**Review by EjoThims — APPROVED** [review]

**Pull request merged**